src/Octoshift/Services/ReclaimService.cs

```diff
@@ -173,7 +173,8 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
                 continue;
             }
 
-            var result = await _githubApi.ReclaimMannequin(githubOrgId, userid, claimantId);
+            var result = skipInvitation ? await _githubApi.ReclaimMannequinsSkipInvitation(githubOrgId, userid, claimantId) : await _githubApi.ReclaimMannequin(githubOrgId, userid, claimantId);
+
 
             HandleResult(login, claimantLogin, mannequin, claimantId, result);
```

Comment:
Will we update this logic to fail fast if org is not EMU? or default to creating invitations?